File: Java/NetBeans/QLTV/src/qltv/CD.java
package qltv;

public class CD extends TaiLieu{
    protected int soTT;
    protected String noiDung;
    
    public CD(String maTL, String tenTL, int viTri, int soTT, String noiDung) {
        super(maTL, tenTL, viTri);
        this.soTT = soTT;
        this.noiDung = noiDung;
    }
    
    public void hienThi() {
        super.hienThi();
        System.out.println("So thu tu: " + soTT);
        System.out.println("Noi dung: " + noiDung);
    }
}

File: C#/DSTL/DSTL/CD.cs
namespace DSTL
{
    public class CD : TAILIEU
    {
        private int soTT;
        private string noiDung;

        public int SoTT { get => soTT; protected set => soTT = value; }
        public string NoiDung { get => noiDung; protected set => noiDung = value; }

        public CD(string maTL, string tenTL, int viTri, int soTT, string noiDung) : base(maTL, tenTL, viTri)
        {
            this.noiDung = noiDung;
            this.soTT = soTT;
        }

        public override void hienThi()
        {
            base.hienThi();
            Console.WriteLine($"So thu tu: {soTT}");
            Console.WriteLine($"Noi dung: {noiDung}");
        }

        public override string loaiTL()
        {
            return "CD";
        }
    }
}
